### ISSUE TITLE
Gridcoin accessibility

### ISSUE THREAD

**Issue by Luly98** [gridcoin-community/Gridcoin-Research#2604]
# Bug Report

**Current behavior**
The GridcoinResearch wallet does not completely support access for screenreader users, though it is not completely inaccessible. The most notable instance of this problem occurs when lists of items are involved: when viewing your addresses, transaction history, saved addresses, and available polls in table view mode, the list found in these sections presents to the screenreader as an empty table, therefore making it impossible to access such information and perform actions like deleting addresses. The same occurs in the vote dialog when voting on a poll that has additional fields: the additional fields present themselves as an empty table, preventing the user from knowing what they are and filling them (the same issue applies to the details dialog of the poll as well). Furthermore, in the account overview screen, recent transactions only appear as the name of the address they pertain to, and no other information is conveyed through the screenreader (e.g., the screenreader only announces the address label, omitting information such as amount, date and time: for example, it would just say "Pool receiving address").

**Expected behavior**
Items listed in tables should be detected as individual elements by the screenreader, so that they can be read and, if appropriate, selected to take action on them. Transactions in the account overview screen should make all their information available to screenreaders.

**Steps to reproduce:**
The issues listed occur on MacOS using the built-in screenreading software, VoiceOver. To reproduce: on a Mac, turn VoiceOver on in System Settings>Accessibility>VoiceOver (or press command+f5), and navigate the Gridcoin app, especially in the areas listed above.

**Gridcoin version**
5.4.0.0-4, downloaded and installed from the gridcoin.us website, no self-building or other custom configuration.

**Machine specs**
- OS: MacOS Ventura 13.0.1
- CPU: Apple M1
- RAM: 8 GB
- Disk size: 245 GB
- Disk Type (HD/SSD): SSD

**Extra information**
I am not a developer, and this is also my first time opening an issue on GitHub: apologies if I didn't fully respect some guidelines, posted to the wrong label (although I think that accessibility should not be treated as a mere feature request), or if this isn't even the correct section to write such comments, or if I did not provide enough information. I tried to be as concise as possible, and there are also a few other accessibility issues I would like to report. I know that not everyone has the same experience in this field, and I would gladly provide more in depth details about how assistive technologies such as screenreaders work for blind users and how we access and use software (or a better explanation of how to reproduce the issues above): I am a blind Gridcoin user, after all. I hope you will consider implementing better accessibility, and I would gladly assist with information or even accessibility testing: I know that using VoiceOver or any other screenreader is not always so obvious and easy for sighted people.

**Comment by jamescowens** [comment]
I am glad you have filed this bug report. I don't think anyone on staff here is experienced in coding or testing for accessibility enabled GUI. I would like to improve what we are doing in this area, however.
